pkg/controller/elasticsearch/nodespec/lifecycle_hook.go

```diff
@@ -29,14 +29,13 @@ set -eux
 # contains IP that is already inactive. Assumes $HEADLESS_SERVICE_NAME and $POD_IP env variables are defined.
 
 # max time to wait for pods IP to disappear from DNS. As this runs in parallel to grace period
-# (defaulting to 30s) after which process is SIGKILLed, it should be set to allow enough time
```

Comment:
We did already override the grace period to 120 sec (now 180), I think the 30sec mention here was a bit misleading.